apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -34,7 +34,18 @@ function load_full_site_editing() {
 	if ( ! is_full_site_editing_active() ) {
 		return;
 	}
+	load_full_site_editing_files_without_verification();
+	Full_Site_Editing::get_instance();
+}
+add_action( 'plugins_loaded', __NAMESPACE__ . '\load_full_site_editing' );
 
+/**
+ * NOTE: In most cases, you should NOT use this function. Please use
+ * load_full_site_editing instead. This function should only be used if you need
+ * to include the FSE files somewhere like a script. I.e. if you want to access
+ * a class defined here without needing full FSE functionality.
+ */
+function load_full_site_editing_files_without_verification() {
```

Comment:
How about naming this in a very super clear way to discourage usages?
I really dig React's `dangerouslySetInnerHTML`, so what do you say about like `dangerously_load_full_site_editing_without_verification`? 😄

Reply:
IMO `load_full_site_editing` sounds dangerous enough. 😄

Reply:
haha yes. I was thinking about that. will do